src/MigrationTools.Clients.AzureDevops.Rest/FieldMaps/FieldLiteralMap.cs

```diff
@@ -1,6 +1,6 @@
-﻿using MigrationTools.Core.Configuration.FieldMap;
+﻿using MigrationTools.Configuration.FieldMap;
```

Comment:
are not the current editorconfig forcing system to be first?

Reply:
Im not sure what you mean

Reply:
using System and related system namespaces should be first in the list. I will add a full editorconfig file that I have been using before and put up a review of that. That will automatically sort the usings correctly, hopefully

Reply:
Any order that they are in is generally visual studios order....

Reply:
Actually I think it adds them in whatever order that they were added.